src/devices/src/legacy/serial.rs

```diff
@@ -96,29 +96,7 @@ pub struct SerialWrapper<T: Trigger, EV: SerialEvents, W: Write> {
     pub input: Option<Box<dyn ReadableFd + Send>>,
 }
 
-impl<W: Write> SerialWrapper<EventFdTrigger, SerialEventsWrapper, W> {
-    fn handle_ewouldblock(&self, ops: &mut EventOps) {
```

Comment:
Make sure you're not throwing away corner-case handling. Some of this code is a direct result of operational issues, study its history and make sure you don't end up repeating it. There be dragons here :dragon_face: